Document how to run apps with a local version of Lightning on the cloud (#16163)
Lightning-AI/lightning: docs/source-app/glossary/index.rst

   :col_css: col-md-12
   :button_link: ../workflows/add_web_ui/glossary_ui.html
   :height: 100

.. displayitem::

>    :header: Using local Lightning on the cloud

Reviewer: ```suggestion
   :header: Using a development branch of Lightning on the Cloud
```

>    :description: Learn how to run apps with your local version of Lightning on the cloud

Reviewer: ```suggestion
   :description: Learn how to contribute to the Lightning framework in the cloud
```
